src/discussions/learners/learner/LearnerFooter.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Icon, OverlayTrigger, Tooltip,
+} from '@edx/paragon';
+import {
+  Edit, Error,
+  QuestionAnswer,
+} from '@edx/paragon/icons';
+
+import messages from './messages';
+import { learnerShape } from './proptypes';
+
+function LearnerFooter({
+  learner,
+  intl,
+}) {
+  const { inactiveFlags } = learner;
+  const { activeFlags } = learner;
+  return (
+    <div className="d-flex align-items-center">
+      <Icon src={QuestionAnswer} className="mx-2 my-0" />
+      <span style={{ minWidth: '2rem' }}>
+        {learner.threads}
+      </span>
+      <Icon src={Edit} className="mx-2 my-0" />
+      <span style={{ minWidth: '2rem' }}>
+        {learner.replies + learner.responses}
+      </span>
+      {Boolean(activeFlags || inactiveFlags)
+        && (
+          <OverlayTrigger
+            overlay={(
+              <Tooltip>
+                <div className="d-flex">
```

Comment:
Without this the UI is broken. 
```suggestion
                <div className="d-flex flex-column align-items-start">
```